### ISSUE TITLE
Selected commit message loses focus after switching to another commit 

### ISSUE THREAD

**Issue by jozydapozy** [desktop/desktop#17896]
### The problem

GitHub Desktop had a really useful feature that has been malfunctioning for a while now (a month?). Let me explain:

Previously, when you were viewing the commit details and selected the commit message (to copy it), then switched to another commit in the left history pane, the commit message of the new commit would automatically be selected.

I'm referring to this element marked in green:

![image](https://github.com/desktop/desktop/assets/119531/30390969-48e1-4a3b-b270-067e8b30900f)

So if you choose another commit on the left, after copying the message, then the message of the other commit would be selected.
Superhandy if you want to copy a bunch of commit-messages to process elsewhere.

This behaviour is broken now since a while. When I now click on another commit, the commit-message loses focus

### Release version

3.3.6

### Operating system

Windows 10

### Steps to reproduce the behavior

1. go to the history-tab
2. select the commit message above the commit where you are
3. switch to another commit
4. see the commit message of that commit is now NOT selected

**Comment by tidy-dev** [comment]
@jozydapozy Wow! That is an interesting use case. I do not believe this was an intentional feature. I suspect the change is a result of the recent work done to make the summary area expandable. That said.. I just tried it on `3.3.6` and it did keep the commit message highlighted. I did notice tho that it is this way if I select from left -> right instead of right -> left. This is likely due to having the entire "selectable" content selected and it is maintaining that "entire selection" on the elements even tho the content is changing. (Note the entire selectable area includes both the commit summary and description -> a change in 3.3.6)

I hope that helps! As for maintaining this feature, it is not something we well likely do unless we were intentionally introducing a feature to make copying commit summaries quickly and I would imagine we would not design it in this way as it is not very discoverable. This kind of functionality has not been a widely requested to my knowledge, with the above work around, I am going to close it. But, we may revisit if we see more interest for it.
